builtin/credential/appgroup/path_app.go

```diff
@@ -0,0 +1,931 @@
+				"app_name": &framework.FieldSchema{
+					Type:        framework.TypeString,
+					Description: "Name of the App.",
+				},
+				"token_ttl": &framework.FieldSchema{
+					Type:        framework.TypeDurationSecond,
+					Description: "Duration in seconds after which the issued token should expire.",
+				},
+			},
+			Callbacks: map[logical.Operation]framework.OperationFunc{
+				logical.UpdateOperation: b.pathAppTokenTTLUpdate,
+				logical.ReadOperation:   b.pathAppTokenTTLRead,
+				logical.DeleteOperation: b.pathAppTokenTTLDelete,
+			},
+			HelpSynopsis:    strings.TrimSpace(appHelp["app-token-ttl"][0]),
+			HelpDescription: strings.TrimSpace(appHelp["app-token-ttl"][1]),
+		},
+		&framework.Path{
+			Pattern: "app/" + framework.GenericNameRegex("app_name") + "/token-max-ttl$",
+			Fields: map[string]*framework.FieldSchema{
+				"app_name": &framework.FieldSchema{
+					Type:        framework.TypeString,
+					Description: "Name of the App.",
+				},
+				"token_max_ttl": &framework.FieldSchema{
+					Type:        framework.TypeDurationSecond,
+					Description: "Duration in seconds after which the issued token should not be allowed to be renewed.",
+				},
+			},
+			Callbacks: map[logical.Operation]framework.OperationFunc{
+				logical.UpdateOperation: b.pathAppTokenMaxTTLUpdate,
+				logical.ReadOperation:   b.pathAppTokenMaxTTLRead,
+				logical.DeleteOperation: b.pathAppTokenMaxTTLDelete,
+			},
+			HelpSynopsis:    strings.TrimSpace(appHelp["app-token-max-ttl"][0]),
+			HelpDescription: strings.TrimSpace(appHelp["app-token-max-ttl"][1]),
+		},
+		&framework.Path{
+			Pattern: "app/" + framework.GenericNameRegex("app_name") + "/wrapped$",
+			Fields: map[string]*framework.FieldSchema{
+				"app_name": &framework.FieldSchema{
+					Type:        framework.TypeString,
+					Description: "Name of the App.",
+				},
+				"wrapped": &framework.FieldSchema{
+					Type: framework.TypeDurationSecond,
+					Description: `Duration in seconds, if specified, enables the Cubbyhole mode. In this mode,
+the UserID creation endpoints will not return the UserID directly. Instead,
+a new token will be returned with the UserID stored in its Cubbyhole. The
+value of 'wrapped' is the duration after which the returned token expires.
+`,
+				},
+			},
+			Callbacks: map[logical.Operation]framework.OperationFunc{
+				logical.UpdateOperation: b.pathAppWrappedUpdate,
+				logical.ReadOperation:   b.pathAppWrappedRead,
+				logical.DeleteOperation: b.pathAppWrappedDelete,
+			},
+			HelpSynopsis:    strings.TrimSpace(appHelp["app-wrapped"][0]),
+			HelpDescription: strings.TrimSpace(appHelp["app-wrapped"][1]),
+		},
+		&framework.Path{
+			Pattern: "app/" + framework.GenericNameRegex("app_name") + "/creds$",
+			Fields: map[string]*framework.FieldSchema{
+				"app_name": &framework.FieldSchema{
+					Type:        framework.TypeString,
+					Description: "Name of the App.",
+				},
+				"user_id": &framework.FieldSchema{
```

Comment:
Why is this here? It will appear in `path-help`. This may become clearer when I get further down the code but it seems like this is unnecessary.